src/quo2/reanimated.cljs

```diff
@@ -68,3 +73,11 @@
 (defn animate-shared-value-with-delay [anim val duration easing delay]
   (set-shared-value anim (with-delay delay (with-timing val (js-obj "duration" duration
                                                                     "easing"   (get easings easing))))))
+
+(defn animate-shared-value-with-delay-repeat [anim val duration easing delay repeat-times]
```

Comment:
consider using full names for vars here.
e.g val -> value 
anim -> animation (?)

Also consider using an object for named params here, in my opinion for functions with more than 2 or 3 parameters it's better to use named params.
e.g
`
(defn animate-shared-value-with-delay-repeat [
{:keys :[anim val duration easing delay repeat-times]}
...`
or at least for some of the parameters

Reply:
Sure, I tried to follow same convention as other functions, but I agree with your suggestion. I will make those changes for this function but we would need to refactor the other ones in other PR so we don't make this one bigger (still more changes to come).